content/applications/hr/time_off.rst

```diff
@@ -150,12 +177,13 @@ Display option section
 
 .. note::
    The only required fields on the time off type form are the name of the :guilabel:`Time Off Type`,
-   the :guilabel:`Approval`, the :guilabel:`Responsible Time Off Officer`, :guilabel:`Take Time Off
-   in`, :guilabel:`Kind of Leave`, and the :guilabel:`Allocation Requests` section.
+   the :guilabel:`Take Time Off In`, and the :guilabel:`Kind of Time Off`. In addition, the
+   :guilabel:`Time Off Requests` and :guilabel:`Allocation Requests` sections must be configured.
```

Comment:
I didn't like how it would look if it was 'fill out the following field:' then there was a note block after a colon. I was also unsure about naming the fields before they were explained, so that's also why it was at the bottom. But I changed that first sentence with a colon, and moved the note beneath it.